src/redux/reducer.type.ts

```diff
@@ -28,6 +28,8 @@ export interface ReduxState {
     notificationIdList: UUID[];
     theme: string;
     nonEvacuatedEnergyNotif: boolean;
+    geoData: any;
```

Comment:
We should definitly define those types

Reply:
i removed them for now,  we need to make @types/powsybl-diagram-viewer to be able to import them in reducer.type.ts